recipes/pyviz/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "pyviz" %}
+{% set version = "0.10.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/p/{{name}}/{{name}}-{{ version }}.tar.gz
+  sha256: 1e0463c31219dcb5018d0bcf01cfc47eed46e7c9970ca1accc52dbb93da234d8
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  entry_points:
+    - pyviz = pyviz.__main__:main
+    
```

Comment:
```suggestion
    skip: True  # [py2k]
    
```

Reply:
@philippjfr I think this is the suggested change if you would like to merge it...

Reply:
Looks okay to me, but I have no idea if it's the correct fix. @jsignell mentioned she had some idea what we should do.

Reply:
This looks right. Try it out!